app/scripts/metamask-controller.js

```diff
@@ -922,6 +925,27 @@ export default class MetamaskController extends EventEmitter {
       state: initState.AccountOrderController,
     });
 
+    const ratesControllerMessenger = this.controllerMessenger.getRestricted({
```

Comment:
Do you think this could cause confusion on first inspection? How can we make the distinction between this controller and `CurrencyRateController` more clear and obvious?